tools/stability-checker/deploy/chart/stability-checker/templates/deploy.yaml

```diff
@@ -30,8 +30,55 @@ spec:
         - name: testing-script-volume
           persistentVolumeClaim:
             claimName: {{ .Values.storage.claimName }}
+        - name: config
+          emptyDir: {}
+        - name: fluentconfig
+          configMap:
+            name: fluentd-config
+        - name: varlog
+          hostPath:
+            path: /var/log
+        - name: varlibdockercontainers
+          hostPath:
+            path: /var/lib/docker/containers
+        - name: google-creds
+          secret:
+            secretName: sa-stability-fluentd-storage-writer
+
+      initContainers:
+        - name: copy-fluentd-config
+          image: alpine:3.9
+          command: ['/bin/sh', '-c', 'cp /config-volume/* /etc/fluentd']
+          volumeMounts:
+          - name: fluentconfig
+            mountPath: /config-volume
+          - name: config
+            mountPath: /etc/fluentd
 
       containers:
+      - name: fluentd
```

Comment:
IMO fluentd should be optional, so I can run stability-checker without fluentd - to keep backward compatibility.